mindmap/static/js/src/requiredModules.js

```diff
@@ -0,0 +1,15 @@
+/*
+We're using requirejs library becuase open edx uses this library to add interal or external modules.
+Check the doc here: https://requirejs.org/
+It's necessary to use this because all moudles out this lib won't work as expected. So this is
+adding our modules to window object that has all modules in the cms, check this:
+https://github.com/openedx/edx-platform/blob/7e23feeb33861c0de3572364cd103c3929bb0588/common/static/js/RequireJS-namespace.js#L3
```

Comment:
```suggestion
Check the doc here: https://requirejs.org/
It's necessary to use this because all modules of this lib won't work as expected. So this is
adding our modules to the window object that has all modules in the CMS, check this:
https://github.com/openedx/edx-platform/blob/7e23feeb33861c0de3572364cd103c3929bb0588/common/static/js/RequireJS-namespace.js#L3
```